src/topology/unit_interval.lean

```diff
@@ -76,7 +75,7 @@ instance : nonempty I := ⟨0⟩
 
 instance : has_mul I := ⟨λ x y, ⟨x * y, mul_mem x.2 y.2⟩⟩
 
-@[simp, norm_cast] lemma coe_mul {x y : I} : ((x * y : I) : ℝ) = x * y := rfl
+@[simp, norm_cast] lemma coe_mul {x y : I} : ((x * y : I) : ℝ) = x * y := Icc.coe_mul x y
```

Comment:
You should keep those `rfl` so that `dsimp` learns about them.

Reply:
These lemmas have all been generalised in `data/set/intervals/instances` to `set.Icc`, and those whose proofs were `rfl` here are also `rfl` in the generalisation so I assume this shouldn't change anything for `dsimp`.  So I'm deleting them from `topology/unit_interval` and now I'm waiting for CI to confirm that this doesn't break anything.

Reply:
Yeah, that sounds better.